diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter3/SortRule.kt

```diff
@@ -80,12 +83,18 @@ class SortRule(private val configRules: List<RulesConfig>) : Rule("sort-rule") {
         sortList: List<ASTNode>,
         nonSortList: List<ASTNode>,
         node: ASTNode) {
+        val spaceBefore = nonSortList.map { astNode ->
```

Comment:
It'll be better to collect all previous sibling till `ENUM_ENTRY`. This way you will be able to store comments while sorting:
```kotlin
enum class Example {
    A /* comment about A */, B,  // comment about B
    C 
}
```

Reply:
What about this? Or are these nodes included in `ENUM_ENTRY`?